api/config.go

```diff
@@ -53,13 +53,14 @@ type Config struct {
 	BaseKey       []byte
 
 	// Swap configs
-	SwapBackendURL          string         // Ethereum API endpoint
-	SwapEnabled             bool           // whether SWAP incentives are enabled
-	SwapPaymentThreshold    uint64         // honey amount at which a payment is triggered
-	SwapDisconnectThreshold uint64         // honey amount at which a peer disconnects
-	SwapInitialDeposit      uint64         // initial deposit amount to the chequebook
-	SwapLogPath             string         // dir to swap related audit logs
-	Contract                common.Address // address of the chequebook contract
+	SwapBackendURL            string         // Ethereum API endpoint
+	SwapEnabled               bool           // whether SWAP incentives are enabled
+	SwapPaymentThreshold      uint64         // honey amount at which a payment is triggered
+	SwapDisconnectThreshold   uint64         // honey amount at which a peer disconnects
+	SwapInitialDeposit        uint64         // initial deposit amount to the chequebook
+	SwapLogPath               string         // dir to swap related audit logs
+	Contract                  common.Address // address of the chequebook contract
+	ChequebookFactoryContract common.Address // address of the chequebook factory contract
```

Comment:
I see that you followed here the style of `Contract` above. I think it would be better to name this variable `SwapChequebookFactory`

Reply:
Same as above I would keep it consistent with `Contract` in this PR and then rename them together in the future.

Reply:
Nevermind, let's just use new names for the new names and only change the old ones in a subsequent PR.